examples/ex18.cpp

```diff
@@ -95,9 +100,11 @@ int main(int argc, char *argv[])
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
-   args.AddOption(&preassemble, "-pa", "--preassemble", "-mf",
-                  "--matrix-free",
-                  "Preassemble weak-divergence assuming F(u) is linear");
+   args.AddOption(&preassembleWeakDiv, "-ea", "--element-assembly-divergence",
+                  "-mf", "--matrix-free-divergence",
+                  "Weak divergence assembly level\n"
+                  "    ea - Element assembly with linearized F\n"
```

Comment:
I wouldn't call replacing the flux `F` with its interpolant `I_h F` "linearized F". "Interpolated F" is probably a better word to use.